build-system/tasks/json-check.js

```diff
@@ -21,6 +21,37 @@ const log = require('fancy-log');
 const through2 = require('through2');
 const {jsonGlobs} = require('../config');
 
+const expectedCaches = ['cloudflare', 'google'];
+
+/**
+ * Fail if caches.json is missing some expected caches.
+ */
+function checkCachesJson() {
+  return gulp.src(['caches.json'])
+      .pipe(through2.obj(function(file) {
+        let obj;
+        try {
+          obj = JSON.parse(file.contents.toString());
+        } catch (e) {
+          log(colors.yellow('Could not parse caches.json. '
+                + 'This is most likely a fatal error that '
+                + 'will be found by checkValidJson'));
+          return;
+        }
+        const foundCaches = [];
+        for (const foundCache of obj.caches) {
+          foundCaches.push(foundCache.id);
+        }
+        for (const cache of expectedCaches) {
+          if (foundCaches.indexOf(cache) == -1) {
```

Comment:
I still think `!foundCaches.includes(cache)` looks better. We can use it here since it's a gulp task, not part of the library :)

Reply:
Ah, I missed that. Ty, done.